### PR TITLE
Change event name to reflect plugin name

### PR DESCRIPTION
**Pull request by mikkamp**
Fixes https://github.com/woocommerce/google-listings-and-ads/issues/840

To avoid reporting issues the plugin name should be used when installing the extension. This PR changes the event name in the onboarding flow from `install_google_ads` to `install_google_listings_and_ads`

### Screenshots
image

image

### Detailed test instructions:
- Open the DevTool console and run `localStorage.setItem( 'debug', 'wc-admin:*' );` to enable tracking debugging mode (make sure all levels are being logged including verbose)
- Confirm we do not have the plugin Google Listing and Ads installed
- Go through the OnBoarding Wizard and complete each step
- At the include free features step, uncheck all extensions except Google Listings and Ads
- Click continue and confirm that the event `wcadmin_storeprofiler_store_business_features_continue` is logged in the console, and the extension name is `install_google_listings_and_ads`

Note: I also had to run the following snippet in the console to be able to expand the output before a page redirect.
```js
window.onbeforeunload = function (e) {
    console.log(e);
    return true;
}
```

### CODE REVIEW

File: client/profile-wizard/steps/business-details/flows/selective-bundle/index.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ class BusinessDetails extends Component {
 				extensionInstallationOptions[ 'mailchimp-for-woocommerce' ],
 			install_mailpoet: extensionInstallationOptions.mailpoet,
 			install_jetpack: extensionInstallationOptions.jetpack,
-			install_google_ads:
+			install_google_listings_and_ads:
```
Comment (joshuatf) [inline client/profile-wizard/steps/business-details/flows/selective-bundle/index.js:106]: If we're switching this to match the plugin name, maybe we can dynamically loop over the plugins to generate this track info.  What do you think?

**Review by joshuatf — COMMENTED** [review]
Thanks for the PR, @mikkamp!  Generally agree with this approach, but just curious if this will cause issues when viewing pre-existing data.

cc @pmcpinto as I know you're also a big tracks fan.

**Comment by mikkamp** [comment]
> just curious if this will cause issues when viewing pre-existing data.

Yes it will need to be taken into consideration that the name changed when checking the tracks. Although it was decided to change it anyways. Here is the slack thread with more background: p1624885679002400-slack-C0187GK80Q1

File: client/profile-wizard/steps/business-details/flows/selective-bundle/index.js (hunk 1 above)
Comment (mikkamp) [reply client/profile-wizard/steps/business-details/flows/selective-bundle/index.js:106]: I'd agree with you. Although I can only speak for the desired tracks event for our plugin. There are other event names that don't match the plugin names, like `mailchimp`. I don't think it's desirable to change those at the moment.

**Review by joshuatf — APPROVED** [review]
Thanks for the clarification, @mikkamp!  This one looks good to merge.  :shipit:

**Pull request merged**